proto/ibc/core/client/v1/tx.proto

```diff
@@ -127,4 +130,22 @@ message MsgUpdateParams {
 }
 
 // MsgUpdateParamsResponse defines the MsgUpdateParams response type.
-message MsgUpdateParamsResponse {}
\ No newline at end of file
+message MsgUpdateParamsResponse {}
+
+// MsgRecoverClient defines the sdk.Msg type to recover a client.
+message MsgRecoverClient {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+  option (cosmos.msg.v1.signer)      = "signer";
+
+  // signer address (it may be the the address that controls the module, which defaults to x/gov unless overwritten).
+  string signer = 1;
```

Comment:
nit but usually the `signer` field is last on our other Msg types

Reply:
#4510